### PR TITLE
Bugfix/15 add missing argument to enrollment query

### PR DESCRIPTION
**Pull request by fschindler**
Depends on #48

### CODE REVIEW

**Comment by fschindler** [comment]
#48 is finally merged and this branch is up to date with `develop`. Ready for review @okwast

File: frontend/components/course/EnrollmentStatus.tsx [hunk 1]
```diff
@@ -17,15 +22,17 @@ interface IProps {
...
-      },
-    }
-  );
+  const { data, loading, error } = useAuthedQuery<
+    CourseWithEnrollment,
+    CourseWithEnrollmentVariables
+  >(COURSE_WITH_ENROLLMENT, {
+    variables: {
+      id: course.Id,
+      authId: keycloak?.subject,
```
Comment (okwast) [inline frontend/components/course/EnrollmentStatus.tsx:33]: What do you think of having a hook that directly gives you the `authId`? I tend to find it tedious to work with the keycloak hook.

**Review by okwast — APPROVED** [review]
Looks good. Thanks for fixing.

Only one idea for a minor improvement (at least for me).

File: frontend/components/course/EnrollmentStatus.tsx (hunk 1 above)
Comment (okwast) [reply frontend/components/course/EnrollmentStatus.tsx:33]: I think we can easily add this in a new PR. So I'm merging this to unblock other things for now.

Comment (fschindler) [reply frontend/components/course/EnrollmentStatus.tsx:33]: Good point, feel free to drop of a PR ;)

**Pull request merged**